app.rb

```diff
@@ -1,8 +1,22 @@
 require 'sinatra/base'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  enable :sessions
+
+  get '/' do
+    erb :index
+  end
+
+  post '/messages' do
+    # Message.create(message: params[:message])
```

Comment:
It looks like you were thinking of creating the message in your POST route above (the commented out bit), which is good. Where would the right place be to retrieve it the message again so that you can display it to the user? You already have all the methods you need to achieve that in your `Message` class!

Reply:
Exactly! Nice one.